syntax/gitrebase.vim

```diff
@@ -26,6 +26,8 @@ syn match   gitrebaseLabel  "\v^l(abel)=>"   nextgroup=gitrebaseName skipwhite
 syn match   gitrebaseReset  "\v^(t|reset)=>" nextgroup=gitrebaseName skipwhite
 syn match   gitrebaseSummary ".*"               contains=gitrebaseHash contained
 syn match   gitrebaseCommand ".*"                                      contained
+syn match   gitrebaseEmpty   " \zs# empty$"        containedin=gitrebaseSummary
+syn match   gitrebaseComment   "# "                containedin=gitrebaseEmpty
```

Comment:
These need `contained` to avoid matching at the top level.

Reply:
Ah! Thanks.  Done.